### ISSUE TITLE
Question about weight importance

### ISSUE THREAD

**Issue by PRamoneda** [Raschka-research-group/coral-pytorch#21]
First of all, thank you so much for your work!

I am trying to use the coral loss function with an unbalanced dataset. Then, I suppose weight importance is for that however the input data have to be sized n_classes -1. I suppose class 0 can not be weighted. Am I right? 

However, class 0 is the smallest one! Do you have some suggestions?

Thank you so much!
